learntools/__init__.py

```diff
@@ -1,3 +1,3 @@
-from . import advanced_pandas, core, deep_learning, gans, machine_learning, python, embeddings, ml_explainability, ml_insights, time_series_with_siraj
```

Comment:
added data_viz_easy in package root's `__init__.py` makes it possible to import stuff from `data_viz_easy`.  Surprised this wasn't an issue for you locally.